Latest version failing to install on BSD/Mac

The bsd version of `install` does not have the `-t` option, so #2543 breaks installs on bsd/macos 
```
ln -sf libtree-sitter.0.0.dylib libtree-sitter.dylib
ln -sf libtree-sitter.0.0.dylib libtree-sitter.0.dylib
sed -e 's|@LIBDIR@|/opt/homebrew/Cellar/tree-sitter/HEAD-49c35b3/lib|;s|@INCLUDEDIR@|/opt/homebrew/Cellar/tree-sitter/HEAD-49c35b3/include|;s|@VERSION@|0.20.10|' \
            -e 's|=/opt/homebrew/Cellar/tree-sitter/HEAD-49c35b3|=${prefix}|' \
            -e 's|@PREFIX@|/opt/homebrew/Cellar/tree-sitter/HEAD-49c35b3|' \
            tree-sitter.pc.in > tree-sitter.pc
install -d '/opt/homebrew/Cellar/tree-sitter/HEAD-49c35b3/lib'
install -m644 -t '/opt/homebrew/Cellar/tree-sitter/HEAD-49c35b3/lib' libtree-sitter.a
install: illegal option -- t
usage: install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 file2
       install [-bCcpSsv] [-B suffix] [-f flags] [-g group] [-m mode]
               [-o owner] file1 ... fileN directory
       install -d [-v] [-g group] [-m mode] [-o owner] directory ...
make: *** [install] Error 64
```

--- a/Makefile
+++ b/Makefile
@@ -61,16 +61,16 @@ install: all
 	    tree-sitter.pc.in > tree-sitter.pc
 
 	install -d '$(DESTDIR)$(LIBDIR)'
-	install -m644 -t '$(DESTDIR)$(LIBDIR)' libtree-sitter.a
-	install -m755 -t '$(DESTDIR)$(LIBDIR)' libtree-sitter.$(SOEXTVER)
+	install -m644 libtree-sitter.a '$(DESTDIR)$(LIBDIR)'/
+	install -m755 libtree-sitter.$(SOEXTVER) '$(DESTDIR)$(LIBDIR)'/
 	ln -sf libtree-sitter.$(SOEXTVER) '$(DESTDIR)$(LIBDIR)'/libtree-sitter.$(SOEXTVER_MAJOR)
 	ln -sf libtree-sitter.$(SOEXTVER) '$(DESTDIR)$(LIBDIR)'/libtree-sitter.$(SOEXT)
 
 	install -d '$(DESTDIR)$(INCLUDEDIR)'/tree_sitter
-	install -m644 -t '$(DESTDIR)$(INCLUDEDIR)'/tree_sitter lib/include/tree_sitter/*.h
+	install -m644 lib/include/tree_sitter/*.h '$(DESTDIR)$(INCLUDEDIR)'/tree_sitter/
 
 	install -d '$(DESTDIR)$(PCLIBDIR)'
-	install -m644 -t '$(DESTDIR)$(PCLIBDIR)' tree-sitter.pc
+	install -m644 tree-sitter.pc '$(DESTDIR)$(PCLIBDIR)'/
 
 clean:
 	rm -f lib/src/*.o libtree-sitter.a libtree-sitter.$(SOEXT) libtree-sitter.$(SOEXTVER_MAJOR) libtree-sitter.$(SOEXTVER)

